content/docs/bdd/_index.md

```diff
@@ -45,7 +45,7 @@ The idea is to make each change small and iterate rapidly, moving back up a leve
 We call these practices _Discovery_, _Formulation_, and _Automation_.
 
 <figure>
-  <img alt="diagram of how the practices fit together" src="../../img/BDD practices diagram (loops).png" width="600px">
+  <img alt="diagram of how the practices fit together" src="../../../static/img/BDD practices diagram (loops).png" width="600px">
```

Comment:
```suggestion
  <img alt="diagram of how the practices fit together" src="../../../img/BDD practices diagram (loops).png" width="600px">
```

Reply:
`static/img/BDD practices diagram (loops).png` is the correct path (see: https://github.com/cucumber/docs.cucumber.io/blob/master/static/img/BDD%20practices%20diagram%20(loops).png).

You can confirm this by going to the `Files changed` tab of this PR and clicking the `Display the rich diff` button where you'll see the expected image.

![image](https://user-images.githubusercontent.com/53531/66349925-d55d8280-e90e-11e9-818f-98dd7284cb29.png)

Reply:
Never mind, I understand the issue now. The way the files are hosted by Netlify gives it a different path. Will fix it.